demos/nordic/nrf52840-dk/common/application_code/main.c

```diff
@@ -416,16 +418,23 @@ static void prvDeleteBonds( void )
 void vApplicationDaemonTaskStartupHook( void )
 {
     uint32_t ulEnabledNetworks;
+    AwsIotTaskPoolInfo_t taskPool = AWS_IOT_TASKPOOL_INFO_INITIALIZER_SMALL;
     /* FIX ME: Perform any hardware initialization, that require the RTOS to be
      * running, here. */
 
     BaseType_t xStatus = pdFALSE;
 
-    if( AwsIotMqtt_Init() == AWS_IOT_MQTT_SUCCESS )
+    xStatus = ( AwsIotTaskPool_CreateSystemTaskPool( &taskPool ) == AWS_IOT_TASKPOOL_SUCCESS );
```

Comment:
System task pool is initialized in System_Init() function here: https://github.com/aws/amazon-freertos/blob/feature/ble-beta/lib/utils/aws_system_init.c#L60
This may lead to multiple initialization

Reply:
System init isn't used on Nordic as it uses old MQTT agent and socket